app/templates/about.html

```diff
@@ -9,6 +9,92 @@ <h2>About the POLDER Federated Search App</h2>
+        <h3>DataONE Repositories</h3>
+        <p>There are many DataONE repositories, but of particular interest to the polar research community are:</p>
+        
+         <div>
+            <a  href="https://arcticdata.io/", rel="noopener", target="_blank">
+                <div class="dataone_repos">
+                <div class="dataone_repos__logo--vertical">
+                <img src="{{ url_for('static', filename='img/arcticdata-logo.png') }}" alt="Arctic Data Committee">
+               
+                </div>
+                </div>
+            </a>
+             <a  href="https://npdc.nl/", rel="noopener", target="_blank">
+                <div class="dataone_repos">
+                <div class="dataone_repos__logo--vertical">
+                <img src="{{ url_for('static', filename='img/npdc-logo.png') }}" alt="Netherlands Polar Data Center">
+               
+                </div>
+                </div>
+            </a>
+             <a  href="https://www.bco-dmo.org/", rel="noopener", target="_blank">
+                <div class="dataone_repos">
+                <div class="dataone_repos__logo--vertical">
+                <img class="bco" src="{{ url_for('static', filename='img/bco-logo.png') }}" alt="BCO-DMO">
+               
+                </div>
+                 <p class="partner__name">BCO-DMO</p>
+                </div>
+            </a>
+             <a  href="usap-dc.org/", rel="noopener", target="_blank">
+                <div class="dataone_repos">
+                    <div class="dataone_repos__logo--vertical">
+                        <img src="{{ url_for('static', filename='img/usap_dc_logo.png') }}" alt="U.S. Antarctic Program Data Center">
+                
+               
+                    </div>
+                        <p class="partner__name">
+                            U.S. Antarctic Program Data Center
+                        </p>
+
+                </div>
+            </a>
+
+        </div>
+         
+                
+                
+
+
+        <h3>Repositories indexed by this app:</h3>
+        
+          <div>
+            <a  href="https://g-e-m.dk/", rel="noopener", target="_blank">
+                <div class="dataone_repos">
+                    <div class="dataone_repos__logo--vertical">
+                        <img src="{{ url_for('static', filename='img/gem.jpg') }}" alt="Greenland Ecosystem Monitoring">
+                        
+                    </div>
+                    <p class="partner__name">Greenland Ecosystem Monitoring</p>
+                </div>
+            </a>
+             <a  href="https://www.bas.ac.uk/", rel="noopener", target="_blank">
+                <div class="dataone_repos">
+                    <div class="dataone_repos__logo--vertical">
```

Comment:
You might try duplicating the corresponding `--horizontal` style here, since these logos are in a landscape format.